lib/src/linden/xstyles.dart

```diff
@@ -21,156 +25,207 @@ class XStyles {
   FontWeight get _weightRegular => FontWeight.w400;
 
   /// Defined by design
-  TextStyle? get appScreenHeadline => _baseTextTheme.headline1?.copyWith(
+  TextStyle get appScreenHeadline =>
+      _baseTextTheme.headline1?.copyWith(
+        color: colors.primaryText,
+        fontWeight: _weightBold,
+        fontSize: 24.0,
+        // height = 32
```

Comment:
❓ Do we still need this commented line ?

Reply:
It’s more of a helper to point that 1.33 height is actually 32 pixels, since in figma height is in pixels.

Reply:
I see, thanks!